multinet/swagger/workspace_aql_tables.yaml

```
@@ -0,0 +1,35 @@
+Create a table from an AQL query
+---
+parameters:
+  - $ref: "#/parameters/workspace"
+  -
+    name: table
```

I prefer to avoid the linebreak here:

```suggestion
  - name: table
```